README.md

```diff
@@ -62,55 +62,64 @@ The WSDL file gives an overview of the values that can be provided, these will v
  * \ID3Global\Identity\PersonalDetails\BirthInfo
  */
 
+$birthday = DateTime::createFromFormat('Y-m-d', '1922-08-20');
 $personalDetails = new \ID3Global\Identity\PersonalDetails();
 $personalDetails
     ->setTitle('Mr')
     ->setForeName('Dworkin')
     ->setMiddleName('John')
     ->setSurname('Barimen')
-    ->setGender(ID3GlobalService::GENDER_MALE)
-    ->setDateOfBirth(1922, 08, 20);
+    ->setGender('male')
```

Comment:
To be clear: There's nothing for you to specifically do here, this is just a note for you :)

Reply:
I thought to use an enumeration for this property, but it would require installing a library (such as `myclabs/php-enum`) unless the package upgraded to PHP 8.1.